module-code/app/securesocial/core/RuntimeEnvironment.scala

```diff
@@ -167,6 +173,7 @@ object RuntimeEnvironment {
       LinkedInProvider.LinkedIn,
       TwitterProvider.Twitter,
       XingProvider.Xing,
+      ChatworkProvider.Chatwork,
```

Comment:
Sorry, this is trivial, but we should use `ChatWork` not `Chatwork`.

Reply:
I do not noticed that official chatwork description is ChatWork. I will fix all occurrences of Chatwork to ChatWork.